src/projectManager.ts

```diff
@@ -184,6 +184,7 @@ export class ProjectManager {
     public static async load(workspaceFolder: WorkspaceFolder) {
         const iProject = new IProject(workspaceFolder);
         if (!this.loaded[workspaceFolder.index]) {
+            await iProject.load();
```

Comment:
why do we load now, instead of waiting for lazy loading? Is this so it is ready to transmit?

Reply:
Yes, I initially did this so that the `buildMap` and `state` are ready to transmit. The reason is because for `IBM i Developer`, our `didOpen` middleware will attempt to retrieve the document ccsid which requires the `buildMap`. Since this is an asynchronous function, it delays the `didOpen` in the language server which results in `completions`, `hover`, `didChange`, etc. from possibly happening before the `didOpen` is done.

If we rely on lazy loading, the very first time we open a document it will have to lazy load the `buildMap` (if it hasn't already) been loaded. In testing this, I think this could be okay since we added the guards to prevent any exceptions and this should only ever happen **one time**. 

@edmundreinhardt Shall I go ahead and revert back to lazy loading?

Reply:
As we discussed, we will keep this change to pre-load the projects